src/main/scala/nl.knaw.dans.easy.stage/dataset/AMD.scala

```diff
@@ -23,7 +23,12 @@ import scala.xml.Elem
 object AMD extends DebugEnhancedLogging {
   type AdministrativeMetadata = Elem
 
-  def apply(depositorId: String, submissionTimestamp: DateTime, state: String): AdministrativeMetadata = {
+  def apply(depositorId: String, submissionTimestamp: DateTime, state: String, remarks: Remarks, stageDatasetVersion: String): AdministrativeMetadata = {
+    val remarksContent =
+      s"""${remarks.privacySensitiveRemark}
+         |
+         |${remarks.messageFromDepositor.getOrElse("")}
+         |""".stripMargin.trim
```

Comment:
This multiline string always has an empty line at the end. Could that be fixed?

Also, if `remarks.messageFromDepositor` returns `Option.empty`, you get 3 empty lines. Perhaps a multiline string isn't the most useful thing here?

I'm thinking something along the lines of:

```scala
remarks.messageFromDepositor.fold(remarks.privacySensitiveRemark)(msg => s"${ remarks.privacySensitiveRemark }\n\n$msg")
```

but there is probably something better than this...

Reply:
The trim looked a much simpler solution to me.

Reply:
I totally overlooked that. Sorry!